xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/layouts/livedata-table.html

```diff
@@ -0,0 +1,90 @@
+<template>
+    <table class="xwiki-livetable">
+
```

Comment:
What's with the empty lines?

Reply:
To have the html less compact and thus more readable imo, but if it's a problem for you I can get rid of it :)